include/functions.h

```diff
@@ -2668,10 +2666,11 @@ void AudioMgr_HandlePRENMI(AudioMgr* audioMgr);
 void AudioMgr_ThreadEntry(void* arg);
 void AudioMgr_Unlock(AudioMgr* audioMgr);
 void AudioMgr_Init(AudioMgr* audioMgr, void* stack, OSPri pri, OSId id, SchedContext* sched, IrqMgr* irqMgr);
-void TitleSetup_GameStateResetContext(void);
-void TitleSetup_InitImpl(GameState* gameState);
-void TitleSetup_Destroy(GameState* gameState);
-void TitleSetup_Init(GameState* gameState);
+
+void Setup_Destroy(GameState* thisx);
+void Setup_Init(GameState* thisx);
+
+
 void Game_UpdateFramerateVariables(s32 divisor);
```

Comment:
```suggestion
void Setup_Init(GameState* thisx);

void Game_UpdateFramerateVariables(s32 divisor);
```
Probably one space is enough